.ci/build-mac.sh

```diff
@@ -1,15 +1,29 @@
 #!/bin/sh -ex
 
 brew update
-brew install llvm@13 sdl2 nasm qt@5 ninja cmake glew git p7zip create-dmg ccache
+brew install llvm@13 sdl2 nasm qt@5 ninja cmake glew git p7zip create-dmg ccache ffmpeg
 
 export MACOSX_DEPLOYMENT_TARGET=11.6
 export CXX=clang++
 export CC=clang
-export Qt5_DIR="/usr/local/opt/qt@5/lib/cmake/Qt5"
-export PATH="/usr/local/opt/llvm@13/bin:/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/Library/Apple/usr/bin"
-export LDFLAGS="-L/usr/local/opt/llvm@13/lib -Wl,-rpath,/usr/local/opt/llvm@13/lib"
-export CPPFLAGS="-I/usr/local/opt/llvm@13/include -msse -msse2 -mcx16 -no-pie"
+
+ARCH_NAME="$(uname -m)"
+if [ "${ARCH_NAME}" = "arm64" ]; then
+  export BREW_PATH=/opt/homebrew/opt
```

Comment:
You can use `brew --prefix` to avoid the if for BREW_PATH, BREW_BIN, BREW_SBIN
```suggestion
  export BREW_PATH=`brew --prefix`/opt
```